core/tests/protractor/explorationFeedback.js

```diff
@@ -50,9 +50,9 @@ describe('ExplorationFeedback', function() {
 
   beforeEach(function() {
     users.createUser('user1@ExplorationFeedback.com',
-                     'creatorExplorationFeedback');
+      'creatorExplorationFeedback');
```

Comment:
See above; ditto below.

I'm not sure what rule was changed in the eslint config to allow this to happen but we should probably remove it.

Reply:
I don't think any rule pertaining to this was changed. Shouldn't we have an indent-by-two in this line?